### ISSUE TITLE
smb_slam still contains compslam and cartographer

### ISSUE THREAD

**Issue by nubertj** [ETHZ-RobotX/smb_common#24]
Hi guys,
Although it is not really my business I was just stumbling about smb_slam package which still mainly consists code from cartographer and compslam. 
Is this planned? or should we replace it with open3d_slam instructions?
I could also help with this.
Best,
Julian

**Comment by mantelt** [comment]
Hi Julian,
thanks for pointing this out. If you think it will be too confusing, we can for sure remove this from the package. 
I thought that it might still be interesting for the participants of the summer school, in case they would like to explore other options. But these old options (cartographer, slam toolbox, compslam), haven't been tested anymore. Thinking about this, I feel like it might create more confusion than it will actually help.  
What do you think?

**Comment by mantelt** [comment]
>  If it is not tested, and students are not supposed to use it, I feel that it would be better to integrate open3d_slam more deeply and remove the legacy code (cartographer and compslam).

The longer I think about that, the more I tend to agree. 

>  I just ran it yesterday to generate a groundtruth map, if you want I can just add a package and launch file here.
What exactly did you use for the groundtrugh map? cartographer? or open3d_slam ?

> Is there any code prepared already how you generated the maps so far?
>  Or is it "just" following the open3d_slam docs?
May @davidoort  or @hibetterheyj  can jump in here. 
I'm not aware of any prepared launch / config files for mapping.

**Comment by nubertj** [comment]
> What exactly did you use for the groundtrugh map? cartographer? or open3d_slam ?
just open3d_slam, nothing special, but we could add a launch and config file, s.t. everything is still correct even if the default on master open3d_slam changes at some point. Plus, students find it automatically if they are looking for smb_slam.

> Maybe @davidoort or @hibetterheyj can jump in here. I'm not aware of any prepared launch / config files for mapping.
Okay, just let me know, I can also quickly prepare a PR and then we can decide (should only be 15 minutes).

**Comment by davidoort** [comment]
@nubertj do you prefer this over having an smb or rss branch in the open3d_slam repo with modified launch files and config files?

**Comment by nubertj** [comment]
i think it would be good for completeness, I can try to add a PR later today.

**Comment by mantelt** [comment]
I agree with @nubertj and @jelavice now. Let's remove the legacy stuff and create a smb specific launch + config file for the open3d_slam in smb_slam

**Comment by nubertj** [comment]
I quickly created a PR last night. Let me know what you think or whether I should update anything.

**Comment by mantelt** [comment]
closing this issue since this is now being addressed with #25
